packages/rpc/src/client.ts

```diff
@@ -108,173 +137,183 @@ export class RPCClient extends EventEmitter {
 
 		/**
 		 * Application used in this client
-		 * @type {?ClientApplication}
 		 */
 		this.application = null;
 
 		/**
 		 * User used in this application
-		 * @type {?Partial<APIUser>}
 		 */
 		this.user = null;
```

Comment:
If they are all just being set to null initially, you could just initialize them as null outside of the constructor (and move the tsdocs there as well)